### PR TITLE
update map tools

### PR DESCRIPTION
**Pull request by jmhauck**

### CODE REVIEW

**Comment by jmhauck** [comment]
@MikeTschudi same for this one...happy to make any updates if you see something.

File: src/components/map-tools/map-tools.tsx [hunk 1]
```diff
@@ -504,13 +543,49 @@ export class MapTools {
...
+   * @param content the widget to display
+   * @param internalExpand when true the widget will be added to an Expand widget
+   *
+   * @protected
+   */
+  protected _getWidget(
+    name: string,
+    content: any,
+    internalExpand: boolean
+  ): undefined {
```
Comment (MikeTschudi) [inline src/components/map-tools/map-tools.tsx:559]: Why `undefined` instead of `void`, please?

**Review by MikeTschudi — COMMENTED** [review]
Looks good; just wondering about something

File: src/components/map-tools/map-tools.tsx (hunk 1 above)
Comment (jmhauck) [reply src/components/map-tools/map-tools.tsx:559]: @MikeTschudi  Not sure if a better approach exists but TS complains if we return void there. When `enableSingleExpand` is true the enabled widgets will be loaded into a single parent expand. Each widget returns a node with the appropriate calcite component that we will render as a part of that expand as is needed for Manager. When that prop is false the appropriate enabled widgets will be loaded and positioned in the UI using the JS api UI logic and we don't actually have any node to control the rendering of.

I believe we could make the parent function `_getMapWidgets` return VNode[] | undefined. Then run `toolOrder.map` if  `enableSingleExpand` is true or `toolOrder.forEach` if it's false. But I couldn't see any advantage of something like that over what we currently have.

**Pull request merged**